pom.xml

```diff
@@ -97,6 +97,7 @@
           <preBuildHookScript>setup.bsh</preBuildHookScript>
           <postBuildHookScript>verify.bsh</postBuildHookScript>
           <addTestClassPath>true</addTestClassPath>
+          <streamLogs>true</streamLogs>
```

Comment:
I have created jira https://issues.redhat.com/browse/JBWS-4228 for this code change.
This allows us to better track changes over time.
Please amend your commit comment to start with "[JBWS-4228]"